pkg/bootstrap/stage.go

```diff
@@ -166,7 +166,19 @@ func extractFromDir(dir, prefix string, img v1.Image, imgName string) error {
 	if err := extract(imgName, tempDir, prefix, r); err != nil {
 		return err
 	}
-	return os.Rename(tempDir, dir)
+	if err := os.Rename(tempDir, dir); err != nil {
```

Comment:
Should check that err is due to target directory already existing. Could be due to permissions, etc.